plugin/apply/apply.go

```diff
@@ -74,8 +75,8 @@ func (g *gen) parseTypes(name string, typs []types.Type) (sig *types.Signature,
 
 func applySig(sig *types.Signature) (last *types.Var, returnFunc *types.Signature) {
 	vs := vars(sig.Params())
-	last = vs[len(vs) - 1]
-	second := types.NewTuple(vs[:len(vs) - 1]...)
+	last = vs[len(vs)-1]
```

Comment:
Been a while since I have done Go, is this gofmted?

Reply:
Yes.

When I first create `apply.go`, the formatter of my IDE (Goland) was not `gofmt`. 
it uses their own formatter which is default.
I have found that later and I have change the setting to use `goimports` (superset of `gofmt`)

Reply:
Come to think of it, CI should have been failed when #65 PR was created.
I missed out on adding the `gofmt` step in CI. I will fix it.

Reply:
Yeah, I also suspected, easy oversight to make, fixing it would be great, thank you

Reply:
Seems CI is still passing, but the change persists?

Reply:
It is expected to pass since `apply.go` is now gofmted. 
If we roll back this PR except CI setting, It will fail.

Reply:
Aha okay great, my bad